src/coreHandler.ts

```diff
@@ -200,10 +201,13 @@ export class CoreHandler {
 			this.core.autoSubscribe('ingestDataCache', {}),
 		])
 		this._subscriptions = this._subscriptions.concat(subs)
+		this.logger.info('Core: Setting up observers for peripheral device commands on ' + this.core.deviceId + '..')
```

Comment:
Wouldn't it make more sense to put these log statements directly in the methods they are intended for? Then, if we decide to keep these statements for the long run we don't have to worry about also creating a log statement everytime we need to call these method (which, I know, is almost never, but still)

Reply:
done